src/core/logging/telegramLogger.ts

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+
+/**
+ *
+ */
+export class TelegramLogger {
+	private token: String;
+	private chatId: String;
+
+	/**
+	 *
+	 */
+	constructor(token: string, chatId: string) {
+		this.token = token;
+		this.chatId = chatId
```

Comment:
missing `;` did you run this through tslint yet?

Reply:
What tslint config do you use? Maybe we should add typescript-eslint and a .eslintrc.cjs to the repository in a later pr

Reply:
An `.eslintrc` file is already in place. 

We probably should just add some extra commands to the `package.json`:
```
"prebuild": "npm run clean && npm run lint && echo \"building...\"",

...

"lint": "eslint . --ext .ts",
"lint:fix": "eslint . --ext .ts --fix",
```